model/src/main/kotlin/Project.kt

```diff
@@ -33,19 +33,14 @@ import java.util.SortedSet
  * meta-data like e.g. the [homepageUrl]. Most importantly, it defines the dependency scopes that refer to the actual
  * packages.
  */
-@JsonIgnoreProperties(value = ["aliases", "purl"], allowGetters = true)
+@JsonIgnoreProperties(value = ["aliases"], allowGetters = true)
```

Comment:
For backwards compatibility it would be better to keep "purl" here, otherwise old results cannot be parsed anymore. But `allowGetters = true` can be removed, it was introduced in 7381820 to allow serialization of `purl`.

Reply:
Good point, thanks, done.